src/searchModal.js

```diff
@@ -132,6 +132,9 @@ export default function searchModalFactory(config) {
                 const route = urlUtil.route('getAll', 'RestResource', 'tao');
                 request(route, classOnlyParams)
                     .then(response => {
+                        if (response.permissions.supportedRights && response.permissions.supportedRights.length > 0) {
```

Comment:
**though:** Is the `permission` property always set in the response?

Reply:
It seems it always set even when `taoDacSimple` is not installed, but I saw that in [other parts of TAO](https://github.com/oat-sa/tao-core/blob/c2e5436443deee674241e77c9530f184105e1328/views/js/provider/resources.js#L141) it is always checked, so added the extra condition here too.